media/css/firefox/whatsnew/whatsnew-modular.scss

```diff
@@ -0,0 +1,288 @@
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// Extra content
+
+.c-block {
+    @include border-box;
+    margin: 0 auto $spacing-2xl;
+    max-width: $content-md - ($layout-md * 2);
+    padding: 0 $layout-md;
+
+    .c-block-title {
+        @include text-display-xs;
+    }
+
+    .c-block-icon {
+        margin-bottom: $spacing-md;
+    }
+}
+
+@media #{$mq-md} {
+    .l-blocks-two {
+        display: flex;
+        margin: 0 auto;
+        max-width: $content-md - ($layout-md * 2);
+        padding: 0 $layout-md;
+
+        .c-block {
+            flex: 1 1 0;
+            padding: 0 $spacing-lg;
+        }
+    }
+}
+
+
+@media #{$mq-lg} {
+    .l-blocks-three {
+        display: flex;
+        margin: 0 auto;
+        max-width: $content-lg;
+
+        .c-block {
+            flex: 1 1 0;
+            padding: 0 $spacing-lg;
+        }
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// FxA Form
+
+.c-signup-container {
+    display: flex;
+    flex-direction: row;
+}
+
+.fxa-form-cta {
+    .fxa-email-form {
+        width: 100%;
+        margin-bottom: $spacing-lg;
+
+        a {
+            color: $color-off-black;
+        }
+    }
+
+    .fxa-email-form-intro {
+        display: none;
+    }
+
+    .fxa-email-field-container {
+        .field {
+            label {
+                @include visually-hidden;
+            }
+
+            input[type="email"] {
+                box-shadow: none;
+                padding: 12px 16px;
+                width: 100%;
+            }
+        }
+
+        .mzp-c-button {
+            width: 100%;
+        }
+    }
+
+    @media #{$mq-sm} {
+        .fxa-email-field-container {
+            @include grid-column-gap($spacing-md);
+            display: grid;
+            grid-template-columns: 2fr 1fr;
+
+            .field {
+                margin-bottom: 0;
+            }
+        }
+    }
+
+    @media #{$mq-md} {
+        .fxa-email-form {
+            display: grid;
+
+            .agreement {
+                @include text-body-sm;
+                grid-row-start: 2;
+                margin-top: $spacing-sm;
+                margin-bottom: 0;
+            }
+        }
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// Dark theme
+
+.t-dark {
+
+    &.content-wrapper {
+        background-color: $color-off-black;
+        color: $color-white;
+    }
+
+    :link {
+        color: $color-white;
+
+        &:hover,
+        &:focus {
+            color: $color-pink-5;
+        }
+
+        &:active,
+        &:focus {
+            background-color: transparentize($color-white, .95);
+        }
+    }
+
+    :visited {
+        color: $color-pink-10;
+
+        &:hover,
+        &:focus {
+            color: $color-white;
+        }
+    }
+
+    .c-page-header {
+        background-color: $color-violet-90;
+    }
+
+    .c-emphasis-box {
```

Comment:
(this is so nitpicky, oh my gosh, _please_ feel free to ignore 😳)
I think for dark theme (or maybe just in general) we should have slightly more contrast between the background and box.

`box-shadow: 0 0 7px rgba(32, 18, 58, 0.28);`

![Screen Shot 2019-07-30 at 1 11 59 PM](https://user-images.githubusercontent.com/30483988/62161796-d13c3580-b2cb-11e9-882d-980b902e10e4.png)
![Screen Shot 2019-07-30 at 1 10 59 PM](https://user-images.githubusercontent.com/30483988/62161797-d13c3580-b2cb-11e9-80c2-eb0198588680.png)

Reply:
Yeah, it's a pretty soft shadow with a wide spread. But I think they want that wide spread to create more elevation... but adding a second shadow with less blur will help sharpen the edge a bit. I'll try it out.